pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -272,6 +272,12 @@ func computeActiveCondition(ctx context.Context, pa *autoscalingv1alpha1.PodAuto
 		pa.Status.MarkScaleTargetInitialized()
 	}
 
+	cond := pa.Status.GetCondition(autoscalingv1alpha1.PodAutoscalerConditionReady)
+	if cond.IsUnknown() && cond.GetReason() == autoscalingv1alpha1.PendingReason {
+		// still at default PA condition, no need to mark anything
+		return
+	}
```

Comment:
Can you comment why we added this? It's not clear why we want to gate setting the Active condition on Ready all of a sudden

Reply:
I'm basically trying to skip the check if its the default condition `with Reason = Pending` that is now set by default when the resource is created, which is different that just `Unknown` that would cause the PA to be set to Inactive

Reply:
yes, it was setting the PA ready to false which made the revision get into the "Initial scale was never achieved" error